doc/Getting-started/index.md

```diff
@@ -54,40 +39,66 @@ You can either install Kaa Sandbox or run it on Amazon EC2.
+For optimal performance we recommend at least 4 GB of RAM, and at least 2 CPUs.
 
-</div><div id="Amazon" class="tab-pane fade" markdown="1">
+5. *\[Optional\]* The default VM networking configuration is NAT.
+In this configuration you will only be able to connect to the Kaa server from your host machine.
+That includes the Kaa client applications, which will only be able to reach your Sandbox server from the host machine.
+
+   Should you wish to deploy client applications to any hardware other than your host machine, or access Kaa web interfaces from your networked devices, the networking mode should be changed to `Bridged Adapter`.
+To configure the networking mode, switch to the `Network` tab in the VM settings.
+Make sure you change the `Promiscuous Mode` to `Allow All` under the `Advanced` spoiler.
+
+   ![VM Networking settings](attach/vm_networking_settings.png)
+
+6. Apply the VM settings and start the virtual machine.
+
+7. Wait until the machine boots up completely and open **127.0.0.1:9080/sandbox** URL in your browser.
+(If you have changed the networking mode to Bridged Adapter, you should rather use **<host\_machine\_ip>:9080/sandbox**.)
 
 <br>
 
-To launch the Kaa sanbox on Amazon Elastic Compute Cloud (Amazon EC2), go through the following steps.
+</div><div id="EC2" class="tab-pane fade" markdown="1">
 
-1. Launch the AMI using the links in the following table:
+<br>
 
-   Amazon EC2 offers a number of [geographic regions](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) for launching the AMI. Factors for choosing a region include: reduce latency, cost, or regulatory requirements.
+> **NOTE:** By running your Kaa Sandbox instance on AWS EC2 you may incur the associated charges.
+> Make sure you have reviewed and understood the current EC2 charging plans before proceeding with the installation.
 
-   To launch an AMI for a specific region, please use the [download](http://www.kaaproject.org/download-kaa/) page.
+To launch the Kaa Sandbox on AWS EC2, do the following:
 
-2. On **Choose an Instance Type** step, choose the appropriate instance type. For optimal performance we recommended that you use at least _m3.large_ instance type, or more powerful.
+1. EC2 offers a number of [geographic regions](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) for launching the [Amazon Machine Images (AMIs)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
+Factors for choosing a region may include latency, cost, or regulatory requirements.
+To launch Kaa Sandbox in your preferred region, please use the [Kaa download page](http://www.kaaproject.org/download-kaa/).
 
-3. On **Configure Instance Details** step, change values in the fields as appropriate or leave default values.
+2. At the `Choose an Instance Type` step, choose the appropriate instance type.
```

Comment:
Good catch. I got confused by some of the guidelines in the doc style page. Will revert.